objects/clients/github.rb

```diff
@@ -24,18 +24,14 @@
 # Github client
 # API: http://octokit.github.io/octokit.rb/method_list.html
 #
-class Github
-  def initialize(config = {})
-    @config = config
-  end
-
-  def client
-    client = if @config['testing']
+class GithubClient
+  def self.new(config = {})
```

Comment:
@mbao01 what's wrong with a constructor? read this: https://www.yegor256.com/2017/11/14/static-factory-methods.html

Reply:
Yes, I read through. Thank you. I will update it